python/pyspark/ml/feature.py

```diff
@@ -512,6 +512,16 @@ class HashingTF(JavaTransformer, HasInputCol, HasOutputCol, HasNumFeatures, Java
     .. versionadded:: 1.3.0
     """
 
+    """
+    Binary toggle to control term frequency counts.
```

Comment:
The comment doc for ```binary``` is unnecessary because it will not exist in the generated Python API doc.